README.md

```diff
@@ -23,14 +29,8 @@ Commands:
   web           Host local web frontend.
 ```
 
-The `hypergit` cli lets you manage hypergit remotes. In order to be able to `git
-clone hypergit://...` URLs, you'll need the `git-remote-hypergit` helper:
-
-```
-npm install -g git-remote-hypergit
-```
-
-**TODO**: install this command with the `hypergit` command! (PRs welcome)
+The `hypergit` cli lets you manage hypergit remotes and enables you to `git
+clone hypergit://...` URLs.
```

Comment:
I removed mentioning `git-remote-hypergit` as @gozala suggested. Maybe we should have a `## How does it work?` section?

Reply:
:+1: